packages/internal/package.json

```diff
@@ -67,6 +67,7 @@
   "devDependencies": {
     "@babel/cli": "7.22.5",
     "@babel/core": "7.22.5",
+    "@babel/plugin-proposal-private-property-in-object": "7.21.11",
```

Comment:
Maybe this should be dependencies given that it's being used in src code?

https://github.com/redwoodjs/redwood/blob/dc3064e4fc9742e5358caa90c2c42a23e3678b29/packages/internal/src/build/babel/common.ts#L70